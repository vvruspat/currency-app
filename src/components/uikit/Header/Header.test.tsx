import { render, screen } from "@testing-library/react";
import { Header } from ".";

test("renders Header", () => {
  render(<Header data-testid="test-header">Test Header</Header>, {});
  const HeaderElement = screen.getByTestId("test-header");
  expect(HeaderElement).toBeInTheDocument();
  expect(HeaderElement).toHaveTextContent("Test Header");
  expect(HeaderElement).toMatchSnapshot();
});
