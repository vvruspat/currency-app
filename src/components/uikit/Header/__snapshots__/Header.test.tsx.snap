// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header 1`] = `
<header
  class="header"
  data-testid="test-header"
>
  <div
    class="title"
  >
    Test Header
  </div>
</header>
`;
